src/mame/heathkit/tlb.cpp

```diff
@@ -1179,6 +1186,11 @@ void heath_superset_tlb_device::crtc_reg_w(offs_t reg, uint8_t val)
 	heath_tlb_device::crtc_reg_w(reg, val);
 }
 
+MC6845_BEGIN_UPDATE(heath_superset_tlb_device::crtc_begin_update)
+{
+	bitmap.fill(rgb_t::black(), cliprect);
+}
```

Comment:
Is this actually necessary?  Shouldn’t it get a call to `heath_superset_tlb_device::crtc_update_row` with `de` set to zero that will result in the rows being cleared if the screen saver is active?

Reply:
Without this, the screen is still not being erased. I'll try to determine why the update row code is not clearing, but right now the `-log` option to mame is not providing the expected logs.

Reply:
@cuavas The reason that `heath_superset_tlb_device::crtc_update_row` is not clearing it, is that the x_count being passed in is 0, since it uses the `m_horiz_disp` variable, which is R1 and that is being set to zero by the firmware.